AppCenterPush/AppCenterPush/MSPush.m

```diff
@@ -326,6 +326,15 @@ - (BOOL)didReceiveRemoteNotification:(NSDictionary *)userInfo fromUserNotificati
   NSObject *title, *message, *customData, *alert;
   NSDictionary *aps = userInfo[kMSPushNotificationApsKey];
 
+  // The service notifications are forwarded to the AppCenter where it will be handled.
+  // Returned after forwarding as the push service will not process those notifications.
+  customData = userInfo[kMSPushNotificationServiceDataKey];
+  customData = ([customData isKindOfClass:[NSDictionary<NSString *, NSString *> class]]) ? customData : nil;
+  if (customData) {
+    [[MSAppCenter sharedInstance] forwardServiceNotification:(NSDictionary<NSString *, NSString *> *)customData];
+    return YES;
+  }
```

Comment:
On line 344-347 we check that push is still enabled. Do we need to worry about that before forwarding?